lib/airtap.js

```diff
@@ -127,6 +115,8 @@ Zuul.prototype.run = function (cb) {
 
     var passed = true
 
+    debug(`begin testing ${self._browsers.length} different browsers`)
```

Comment:
Did you add this for the refactoring, or is it something you want to keep?

Reply:
I'd like to keep it :)

Reply:
Actually, I think we should have more debug statements in the code.